### PR TITLE
Fix RTL1

### PR DESCRIPTION
**Pull request by ricardopereira**

### CODE REVIEW

**Comment by mattheworiordan** [comment]
Seems good to me, but not sure about the error, will have to leave that to @tcard

File: Spec/RealtimeClientChannel.swift [hunk 1]
```diff
@@ -70,11 +67,11 @@ class RealtimeClientChannel: QuickSpec {
                     }
```
Comment (tcard) [inline Spec/RealtimeClientChannel.swift]: I think line 49 above (can't comment on it) also needs a `toEventually`, or a wait until for the `channel2.attach()` callback.

Comment (tcard) [inline Spec/RealtimeClientChannel.swift]: Also, the `subscribe` on line 52 should be moved above the `channel2.attach()`, to make sure it happens before it is attached.

Comment (ricardopereira) [inline Spec/RealtimeClientChannel.swift]: Done 60b1ef1. Thanks.

**Comment by tcard** [comment]
LGTM

**Pull request merged**